CMP1903_A1_2324/Program.cs

```diff
@@ -10,10 +10,30 @@ internal class Program
     {
         static void Main(string[] args)
         {
+            bool valid = false;
+            while (!valid)
+            {
+                Console.WriteLine("Do you want to run the test: Y or N");
+                try
+                {
+                    string InputtedValue = Console.ReadLine();
+                    if (InputtedValue == "Y" || InputtedValue == "X")
+                    {
+                        
```

Comment:
code wont progress past this point so i am unable to verify the rest of the code as working